packages/cli/src/lib/deploy/sequence.ts

```diff
@@ -13,22 +13,22 @@ export interface SequenceResult {
   }[];
 }
 
-interface SequenceArgs {
+interface DeployJobArgs {
   name: string;
-  steps: Step[];
+  steps: DeployStep[];
   config: Record<string, unknown>;
   printer: GluegunPrint;
 }
 
-export class Sequence {
+export class DeployJob {
```

Comment:
This could implement the `DeployJob` interface from the generated `polywrap.deploy.yaml` manifest types.

Reply:
Same comment about type exporting applies here